documentation/IDTA-01002-3/modules/ROOT/pages/specification/interfaces-payload.adoc

```diff
@@ -708,6 +726,23 @@ e|outputArguments |Output argument |link:https://admin-shell-io.github.io/aas-sp
 e|inoutputArguments |InOutput argument |link:https://admin-shell-io.github.io/aas-specs-antora/IDTA-01001/v3.1/spec-metamodel/submodel-elements.html#OperationVariable[OperationVariable] |0..*
 |===
 
+
+[#operation-result-value-only]
+===== OperationResultValueOnly
+
+[.table-with-appendix-table]
+[cols="25%,40%,25%,10%"]
+|===
+h|Class Name 3+e|[[OperationResultValueOnly]]OperationResultValueOnly
+h|Explanation 3+a|The operation’s invocation result object in the ValueOnly notation.
+h|Inherits from 3+|
+h|semanticId 3+|`\https://admin-shell.io/aas/API/DataTypes/OperationResultValueOnly/3/1`
+
+h|Attribute h|Explanation h|Type h|Card.
+e|outputArguments |Output argument |link:https://admin-shell-io.github.io/aas-specs-antora/IDTA-01001/v3.1/mappings/mappings.html#value-only-serialization-in-json[Object in ValueOnly notation] |0..*
```

Comment:
Similar to the in(out)put parameters, the type needs to be more clearly defined than `Object`, either `OperationVariable` or `SubmodelElement`.